actors/verifreg/src/lib.rs

```diff
@@ -751,8 +759,11 @@ where
                 serialization; "failed to marshal remove datacap request: {}", e)
     })?;
 
+    let mut payload = SIGNATURE_DOMAIN_SEPARATION_REMOVE_DATA_CAP.as_bytes().to_vec();
+    payload.append(&mut b.to_vec());
```

Comment:
```suggestion
    let payload = [SIGNATURE_DOMAIN_SEPARATION_REMOVE_DATA_CAP, b.bytes()].concat();
```

(assuming you make the change above)

Reply:
(Avoids a clone, vector resize, and is slightly cleaner)